notebooks_tsqr/NightLog.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #64.            to = (timeout or self.timeout)
</pre><p>I suggest <code>timeout =</code> but also, what happens if they are both set, the first entry overrules?  Maybe we should prefer <code>timeout = max(timeout, self.timeout)</code> to select the larger of the two? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>